test/extended/fixtures/ldap/ldapserver-deploymentconfig.json

```diff
@@ -58,7 +58,14 @@
             "securityContext": {
               "capabilities": {},
               "privileged": false
-            }
+            },
+            "readinessProbe": {
+              "exec": {
+                "command": ["bash", "-c", "\"ldapsearch -x -b dc=example,dc=com | grep organization\""]
```

Comment:
surprised the quoting on the third arg is needed... is it not treated as a single arg to bash?